cmd/servicelog/post.go

```diff
@@ -38,21 +41,17 @@ var postCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 
 		parseUserParameters() // parse all the '-p' user flags
+		readFilterFile()      // parse the ocm filters in file provided via '-f' flag
+		readTemplate()        // parse the given JSON template provided via '-t' flag
 
-		readTemplate() // parse the given JSON template provided via '-t' flag
-
-		// For every '-p' flag, replace its related placeholder in the template
-		for k, v := range templateParams {
-			replaceFlags(userParameterValues[k], "", userParameterNames[k], userParameterNames[k], "p", v)
+		// For every '-p' flag, replace its related placeholder in the template & filterFiles
+		for k := range userParameterNames {
+			replaceFlags(userParameterNames[k], userParameterValues[k])
 		}
 
-		// Check if there are any remaining placeholders in the template that are not replaced by a parameter
-		checkLeftovers()
-
-		dir := tempDir()
-		defer cleanup(dir)
-
-		newData := modifyTemplate(dir)
+		// Check if there are any remaining placeholders in the template that are not replaced by a parameter,
+		// excluding '${CLUSTER_UUID}' which will be replaced for each cluster later
+		checkLeftovers([]string{"${CLUSTER_UUID}"})
```

Comment:
does that still work in the case of posting to a single cluster?

Reply:
Yep, I've confirmed that servicelogs can still be posted to a single cluster: 
```
[~/workspace/osdctl]
[tnierman@tnierman] >> ./dist/osdctl_linux_amd64/osdctl servicelog post -t https://raw.githubusercontent.com/openshift/managed-notifications/master/osd/cluster_has_gone_missing.json -p CLUSTER_UUID=tnierman
INFO[0000] The following clusters match the given parameters: 
Name                ID                                 State               Version             Cloud Provider      Region
tnierman            1mgrgm293o9h83jfove9ndv5i1p5ntis   installing                              aws                 us-west-1

INFO[0000] The following template will be sent:         
{
  "severity": "Error",
  "service_name": "SREManualAction",
  "cluster_uuid": "${CLUSTER_UUID}",
  "summary": "Action required: delete unregistered cluster",
  "description": "Your cluster requires you to take action because it is no longer registering with Red Hat OpenShift Cluster Manager. Possible causes include stopping instances or a networking misconfiguration. If you intended to terminate this cluster then please delete the cluster. Otherwise file a support request.",
  "internal_only": false,
  "event_stream_id": ""
}
Continue? (y/N): y
INFO[0002] Message has been successfully sent to 2d3d511b-e3d0-41e5-be8b-c1611d89dd1e 
```